sources/bloodhound/src/bin/bottlerocket-checks/checks.rs

```diff
@@ -577,15 +577,17 @@ pub struct BR03030100Checker {}
 
 impl Checker for BR03030100Checker {
     fn execute(&self) -> CheckerResult {
-        let result = check_file_contains!(
-            PROC_MODULES_FILE,
-            &["sctp"],
-            "unable to parse modules to check for sctp",
-            "sctp is currently loaded"
-        );
+        let mut result = CheckerResult::default();
 
-        // Check if we need to continue
-        if result.status == CheckStatus::FAIL {
+        // Make sure sctp isn't already loaded
+        if let Ok(found) = look_for_string_in_file(PROC_MODULES_FILE, "sctp") {
```

Comment:
I assume your plan is to switch this to `look_for_word_in_file` once #3562 merges. The change looks good to me otherwise.

Reply:
Agree - I will wait for that one to merge to add the macro, then update this one to take advantage of that.

Reply:
3562 has merged. Rebased and updated this one to take advantage of the `look_for_word_in_file` check.